docs/dev-environments/ethereum/besu/docker-compose-truffle.yaml

```diff
@@ -24,7 +24,7 @@ services:
         - ./my_project:/my_project
         - $TCF_HOME/sdk/avalon_sdk/ethereum/contracts/proxy-model:/contracts
         - $TCF_HOME/sdk/avalon_sdk/ethereum/truffle_artifacts:/truffle_artifacts
-      container_name: truffle-envt-setup
+      container_name: truffle-envt-besu-setup
       command: "bash -c \"\
           cd my_project && \
           truffle init && \
```

Comment:
I've same comments as in https://github.com/hyperledger/avalon/pull/327/files#r393592576